sample_apps/go/query-common.go

```diff
@@ -26,8 +26,11 @@ func write(f *os.File, s string) {
 	}
 }
 
-func processScalarType(data *timestreamquery.Datum) string {
-	return *data.ScalarValue
+func processScalarType(datum *timestreamquery.Datum) string {
+	if datum.NullValue != nil && *datum.NullValue {
+		return "NULL"
```

Comment:
Can you give me an ELI5 of how this “NULL” is used ? Why isn’t it a type as nil or null and instead we use a string representation of it ?

Reply:
We are displaying a query result as a table. So we need this to be a string.

Reply:
Gotcha